app/scripts/modules/kubernetes/src/pipeline/stages/runJob/runJobStage.js

```diff
@@ -36,6 +36,12 @@ module.exports = angular.module('spinnaker.kubernetes.pipeline.stage.runJobStage
     this.stage.cloudProvider = 'kubernetes';
     this.stage.application = $scope.application.name;
 
+
+    if (this.stage.container && !this.stage.containers) {
```

Comment:
I haven't followed the backend changes for this PR. Will Clouddriver still work if someone has a `stage.container`?

Reply:
Yep! I made those changes yesterday. I tried to keep is non-breaking: https://github.com/spinnaker/clouddriver/pull/2135

Reply:
Awesome, thanks.